### DISCUSSION TITLE
Proctoring with SEB client

### DISCUSSION THREAD

**Discussion by ghost** [SafeExamBrowser/seb-server#46] [General]
Dear SEB team, 

I am interested in using live proctoring. I tried to read all the available information on the web regarding this feature, but I didn't get it well. If I understand it correctly, using Jitsi Meet is a better option compared to Zoom. 
I have a few questions. 
Is it possible to run live proctoring only with the SEB client and without the SEB server? Are there some guidelines to follow to set it appropriately? I found only instructions for the SEB server. 

Is the feature for head control available also for Zoom or only for Jitsi Meet?

Thank you very much for your reply.

**Comment by anhefti** [comment]
Hi and sorry for the late answer.

> Is it possible to run live proctoring only with the SEB client and without the SEB server

We are working hard to make it possible to use live proctoring service for SEB with or without SEB Server in the future with Zoom or Jitsi Meet as planed so far. Currently it is not easily possible to use live proctoring without SEB Server. Please refer also to [this discussion](https://github.com/SafeExamBrowser/seb-server/discussions/43#discussioncomment-1740811) on the same subject. 

> Is the feature for head control available also for Zoom or only for Jitsi Meet?

As far as I know it is only available for the latest SEB iOS and SEB Mac versions of Jitsi Meet proctoring integration. I try to clarify this. Or please refer to the appropriate SEB feature lists to get more information on that.

Best.
